extensions/podman/src/extension.ts

```diff
@@ -79,7 +80,8 @@ export type MachineInfo = {
 
 async function updateMachines(provider: extensionApi.Provider): Promise<void> {
   // init machines available
-  const machineListOutput = await execPromise(getPodmanCli(), ['machine', 'list', '--format', 'json']);
+  const machineListOutput = (await extensionApi.process.exec(getPodmanCli(), ['machine', 'list', '--format', 'json']))
+    .stdout;
```

Comment:
I think destructuring is easier to read
```suggestion
  const {stdout: machineListOutput} = await extensionApi.process.exec(getPodmanCli(), ['machine', 'list', '--format', 'json']);
```

Reply:
updated